ide-common/src/main/java/org/digma/intellij/plugin/refreshInsightsTask/RefreshService.kt

```diff
@@ -33,7 +35,7 @@ class RefreshService(private val project: Project) {
     private val isGeneralRefreshButtonEnabled = AtomicBoolean(true)
 
     companion object {
-
+        private val logger = Logger.getInstance(RefreshService::class.java)
         @JvmStatic
```

Comment:
why inside companion class and not as class member?

Reply:
we wanted to log the get instance. so we had to put it in the companion object